docker-compose.yml

```diff
@@ -1,31 +1,31 @@
 version: '3'
 
 services:
-  postgres:
+  db:
     image: postgres:latest
     environment:
       POSTGRES_DB: ${DATABASE_NAME}
       POSTGRES_USER: ${DATABASE_USER}
       POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
     volumes:
-      - datablitz_db_data:/var/lib/postgresql/data
+      - db:/var/lib/postgresql/data
```

Comment:
Can be cool to have a less generic name to be easy to find in local when we do a `docker volume ls`

Reply:
@Jerome-Celle explain to me that the name is prefixed by default, so it's all great!